packages/ppx_camlrack/ppx_camlrack.0.9.0/opam

```diff
@@ -0,0 +1,45 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "PPX for matching S-Expressions"
+description: """
+'PPX-Camlrack' provides an augmented match expression that
+transforms written S-expressions into their Camlrack-compatible types. This
+allows for easily working with S-expressions without extra typing."""
+maintainer: ["Pierce Darragh <pierce.darragh@gmail.com>"]
+authors: ["Pierce Darragh <pierce.darragh@gmail.com>"]
+license: "MIT"
+tags: ["S-Expressions" "parsing" "matching" "ppx" "extension"]
+homepage: "https://github.com/pdarragh/camlrack"
+doc: "https://github.com/pdarragh/camlrack"
+bug-reports: "https://github.com/pdarragh/camlrack/issues"
+depends: [
+  "dune" {>= "2.9"}
+  "ocaml" {>= "4.12"}
+  "ppxlib" {>= "0.23.0"}
+  "camlrack" {>= "0.9.0"}
```

Comment:
```suggestion
  "camlrack" {= version}
```
For ease of maintenance, I would suggest to bind the two version numbers and always release the two packages together